src/main/java/sekta/platform/core/entity/EntityModel.java

```diff
@@ -0,0 +1,30 @@
+package sekta.platform.core.entity;
+
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.MappedSuperclass;
+
+/**
+ * Created by Retro on 19.05.2016.
+ */
+
+@MappedSuperclass
+public abstract class EntityModel {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
```

Comment:
Not all Entities has column `id`. Some entities has a complex primary keys, usually programmers doesn't use Abstraction for entities